.rubocop.yml

```diff
@@ -20,10 +23,22 @@ AllCops:
     - "Guardfile"
     - "config/spring.rb"
     - "config/application.rb"
+    - 'generated/**/*.rb'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'script/**/*'
+    - 'bin/{rails,rake}'
+    - !ruby/regexp /old_and_unused\.rb$/
+    - "lib/**/*"
```

Comment:
@basti I ignored lib because there were errors in the cucumber.rake file that was generated by gem, so I thought that I should not correct them. Now I fixed it.